apps/ehr/src/features/visits/shared/components/exam-tab/ControlledCheckboxSelect.tsx

```diff
@@ -1,39 +1,46 @@
 import { Box, ListItemText, MenuItem, Select } from '@mui/material';
-import { FC, useEffect, useState } from 'react';
+import { FC, useState } from 'react';
 import { useExamObservations } from 'src/features/visits/telemed/hooks/useExamObservations';
 import { StatelessExamCheckbox } from './StatelessExamCheckbox';
 
 export type ControlledCheckboxSelectOptionType = { label: string; name: string; description?: string };
 
 type ControlledCheckboxSelectProps = {
   label: string;
+  name: string;
   options: ControlledCheckboxSelectOptionType[];
 };
 
 export const ControlledCheckboxSelect: FC<ControlledCheckboxSelectProps> = (props) => {
-  const { label, options } = props;
-
-  const { value: fields, update, isLoading } = useExamObservations(options.map((option) => option.name));
+  const { label, name, options } = props;
 
+  const params = [name].concat(options.map((option) => option.name));
+  let { value: fields } = useExamObservations(params);
+  // appointments made before https://github.com/masslight/ottehr/issues/4055 is ready might have some undefined fields
+  fields = fields.filter((field) => field !== undefined);
+  const { update, isLoading } = useExamObservations(params);
```

Comment:
Can we unite this two `useExamObservations`? They have the same params so I think it should work
```suggestion
  let { value: fields, update, isLoading } = useExamObservations(params);
  // appointments made before https://github.com/masslight/ottehr/issues/4055 is ready might have some undefined fields
  fields = fields.filter((field) => field !== undefined);
```